src/camera.ts

```diff
@@ -310,10 +304,9 @@ class Camera extends Element {
         this.entity.camera.renderTarget = renderTarget;
         this.entity.camera.camera.horizontalFov = width > height;
 
-        // create pick mode render target
-        this.pickModeColorBuffer = createTexture(width, height, pixelFormat);
+        // create pick mode render target (reuse color buffer)
         this.pickModeRenderTarget = new RenderTarget({
-            colorBuffer: this.pickModeColorBuffer,
+            colorBuffer: colorBuffer,
```

Comment:
You can shorthand this to just `colorBuffer`.

Reply:
typically I try to avoid it .. as then sometimes I rename the local variable and wonder why the code no longer works.

Reply:
done, and elsewhere